waspc/data/Generator/templates/react-app/src/config.js

```diff
@@ -1,6 +1,6 @@
 import { stripTrailingSlash } from "./universal/url";
 
-const apiUrl = stripTrailingSlash(process.env.REACT_APP_API_URL) || 'http://localhost:3001';
+const apiUrl = stripTrailingSlash(import.meta.env.REACT_APP_API_URL) || 'http://localhost:3001';
```

Comment:
Ideally, this variable is something like `WASP_API_URL` since the underlying tech doesn't matter.

I would rethink this in the future (here's an issue for this: https://github.com/wasp-lang/wasp/issues/1036), but for now I went with minimising PR impact and keeping it backwards compatible so the upgrade path is cleaner.

Please comment your preferences on the issue I linked :)

Reply:
Yup, we should certainly look into changing this! It was using REACT_APP_ because it was easier to do it that way at the moment I believe, but if we can get away from that, that would be great.
@infomiho I do suggest we go after this relatively soon, since this is a pretty good moment to tackle it, when we are doing breaking changes anyway and removing CRA.